kitsune/sumo/jinja2/base.html

```diff
@@ -32,6 +32,10 @@
     {% stylesheet style %}
   {% endfor %}
 
+  {% if waffle.switch('fxa-banner-test') and request.LANGUAGE_CODE in settings.FXA_BANNER_LANGUAGES %}
```

Comment:
NIT: since the waffle switch check hits the DB it'd be more efficient to check it after the language check so that it's checked less often.